activesupport/CHANGELOG.md

```diff
@@ -17,6 +17,10 @@
 
     *Jean Boussier*
 
+*   Add `ActiveSupport::Notifications::EventAssertions` module to help with testing events.
+
+    *Nicholas La Roux*, *Yishu See*, *Sean Doyle*
+
```

Comment:
Given the extremely helpful review that's greatly shaped this contribution, I think it makes sense to include you hear as well @seanpdoyle. :pray:

Reply:
Changelog entries need to be at the top of the file.